src/com/sun/jna/CallbackReference.java

```diff
@@ -69,6 +67,20 @@
             throw new Error("Error looking up CallbackProxy.callback() method");
         }
     }
+    
+    private static final Class<Callback> DLL_CALLBACK_CLASS;
+    
+    static {
+        if (Platform.isWindows()) {
+            try {
+                DLL_CALLBACK_CLASS = Class.forName("com.sun.jna.win32.DLLCallback");
+            } catch(Exception e) {
```

Comment:
Please only catch Exception if you have a very good reason in this case `ClassNotFoundException` is appropriate.  Please also don't swallow the cause of the error, so pass the exception to the error as cause.